client/components/Reviews.jsx

```diff
@@ -4,10 +4,10 @@ import Review from './Review.jsx';
 const Reviews = (props) => {
```

Comment:
Another way to do props, would be use es6 de-structuring 

`{ reviews, stars }`

Lets you avoid lines 6 & 7